picocom.c

```diff
@@ -1202,6 +1213,9 @@ loop(void)
 			} while ( n < 0 && errno == EINTR );
 			if ( n <= 0 )
 				fatal("write to term failed: %s", strerror(errno));
+			if ( opts.lecho && opts.log_filename )
+				if ( writen_ni(log_fd, tty_q.buff, sz) < sz )
```

Comment:
Talking about picocom.c:1194, I think that `buff_rd` is correct. When I change this to `tty_q.buff`, then I got wrong content in the logfile.